chain-signatures/node/src/metrics.rs

```diff
@@ -14,26 +14,26 @@ pub(crate) static NODE_RUNNING: Lazy<IntGaugeVec> = Lazy::new(|| {
     .unwrap()
 });
 
-pub(crate) static NUM_SIGN_REQUESTS: Lazy<IntGaugeVec> = Lazy::new(|| {
-    try_create_int_gauge_vec(
+pub(crate) static NUM_SIGN_REQUESTS: Lazy<CounterVec> = Lazy::new(|| {
```

Comment:
What's the difference between CounterVec and IntGaugeVec?

Reply:
You can not go down with counter. Also, there are some inconsistencies with the data. We are trying to find the reason.